### ISSUE TITLE
Problem with table collum

### ISSUE THREAD

**Issue by Cu8eR** [megasnort/fork-cms-module-elo-rating#17]
check http://schakenopviso.megasnort.com/partijen

collums have different width

**Comment by megasnort** [comment]
I actually left it that way on purpose: I provided a basic template with just simple tables. Tables work everywhere and probably look pretty ok in every existing theme.

To let the columns have the same width, I would have to include:
- Javascript, for styling
- Extra classes
- ... or some other really specific way of styling
which would all be incompatible with most (probably all) existing themes.

In other words, I leave it the one using the Module, to provide styling to his/or her liking if she/he is not satisfied with the different widths.

**Comment by Cu8eR** [comment]
@megasnort - what about to add width in % for every collumn?

**Comment by megasnort** [comment]
I added width's to the header in the html, so I didn't need to add extra css.
